### ISSUE TITLE
Support multiple formatters

### ISSUE THREAD

**Issue by garygreen** [stevebauman/log-reader#5]
Monolog allows you to store the log in various formatters. It would be nice to be able to specify a format, along with a manual source of input:

```php
LogReader::formatter('Monolog/Formatter/JsonFormatter')->get();
```

**Comment by stevebauman** [comment]
Agreed, will definitely implement, or if you're interested I'll accept a PR as well.
